gef.py

```diff
@@ -8246,7 +8246,11 @@ def context_source(self):
 
         file_base_name = os.path.basename(symtab.filename)
         breakpoints = gdb.breakpoints() or []
-        bp_locations = [b.location for b in breakpoints if file_base_name in b.location]
```

Comment:
To preserve the code style something like this would also be possible here:

```diff
- bp_locations = [b.location for b in breakpoints if file_base_name in b.location]
+ bp_locations = [b.location for b in breakpoints if b.location and file_base_name in b.location]
```

Or did you intentionally change it because you are of the opinion that this is confusing?

Reply:
I didn't write in that way is I'm not sure if there is any runtime optimization which changes the execution order as a syntactic sugar.

Reply:
I agree with @theguy147 the suggested syntax is not very pythonic, 

```python
        if not b.location == None:
```

Reply:
Fixed in commit 2eb45e7d10b7b5c881996d9818038db090190457.